### PR TITLE
Doc: instructions to extract files

### LINKED ISSUE #353: Add instructions for extracting game asset files
* installation from disk
* mentioned supported patches (1.2)

### PR DESCRIPTION
**Pull request by captnfab**
Fixes #353

Note:

- not sure if we should mention the fan site
- I think it's nice to mention that the unofficial patch won't break things (at least for people who already have it installed)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -85,6 +85,62 @@ grammar.
...
+1. Install the game from the installer, remember the installation directory
+(`C:\\Program Files (x86)\\Lionhead Studios\\Black & White\\` is the default)
+2. Install the official patch 1.10 (in the same directory)
+3. Install the official patch 1.2 (in the same directory)
+
+### On Linux
+
+If you run on Linux, you can install the game with Wine.
+
+1. Install `wine`
```
Comment (bwrsandman) [inline readme.md:119]: Provide link to the winedb page for black & white

File: readme.md [hunk 2]
```diff
@@ -85,6 +85,62 @@ grammar.
...
+   `winetricks first`)
+
+All the files should be installed in the following directory:
+`~/.wine/drive_c/Program Files (x86)/Lionhead Studios Ltd/Black & White`
+
+### Getting the assets
+
+The folders you're interested in are the `Data` and `Scripts` folders in the
+installation directory. You can use them from there or make a copy to the
+location of your choice.
```
Comment (bwrsandman) [inline readme.md:137]: There's more than that. We want the user to keep track of the folder that contains both those files.

**Review by bwrsandman — CHANGES_REQUESTED** [review]

File: readme.md [hunk 3]
```diff
@@ -85,6 +85,62 @@ grammar.
...
+
+
+# License
+
 openblack is released as open source software under the [GPL v3](https://opensource.org/licenses/gpl-3.0.html)
 license, see the [license](./license) file in the project root for the full license text.
+
+Copyright of the original game assets belongs to Lionhead Studios. To use them,
+you are legally required to own a copy of Black & White original game.
```
Comment (handsomematt) [inline readme.md:146]: We don't need to specify this

File: readme.md (hunk 1 above)
Comment (captnfab) [reply readme.md:120]: Ok, I added the link to the winehq appdb page.

File: readme.md (hunk 2 above)
Comment (captnfab) [reply readme.md:138]: I added some precision. If it's still not sufficient, please be more specific on what I should clarify.

File: readme.md (hunk 3 above)
Comment (captnfab) [reply readme.md]: Ok, removed.

File: readme.md [hunk 4]
```diff
@@ -85,6 +85,73 @@ grammar.
...
+## Installation
+
+### On Windows
+
+1. Install the game from the installer, remember the installation directory
+(`C:\\Program Files (x86)\\Lionhead Studios\\Black & White\\` is the default)
+2. Install the official patch 1.10 (in the same directory)
+3. Install the official patch 1.2 (in the same directory)
+
+### On Linux
```
Comment (bwrsandman) [inline readme.md:115]: The instructions are the same for MacOS, please add that.

**Review by bwrsandman — CHANGES_REQUESTED** [review]

File: readme.md (hunk 4 above)
Comment (captnfab) [reply readme.md:115]: @bwrsandman : where is the wine directory on macos? is it in ~/.wine/ too?

Comment (captnfab) [reply readme.md]: According to some docs, it is, I pushed the new version.

**Review by bwrsandman — APPROVED** [review]
Good first contribution!

**Pull request merged**